python/cudf/cudf/core/series.py

```diff
@@ -155,7 +155,9 @@ def values(self):
         if len(self) == 0:
             return cupy.asarray([], dtype=self.dtype)
 
-        return cupy.asarray(self.to_gpu_array())
+        return cupy.asarray(
+            numbautils.PatchedNumbaDeviceArray(self.to_gpu_array())
```

Comment:
I wonder if we should add a version check or at least a note/gh issue that this is a temp solution while copy updated is available?

Reply:
actually the way I did in cuML is to detect if there is a version conflict, if there is then use the `numbautils.PatchedNumbaDeviceArray`, if not then use the regular, no error raised. So neither CI nor dev environments break (and CI right now amusingly tests both 0.45.1 and 0.46, which is actually nice). Also the check for cupy is to check `<7.0`, since anything 6.3, 6.4 or 6.5 will break, but tbh I'm fine with the comment and github issue